src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,29 @@
+package calculator
+
+class Calculator(expressionParser: ExpressionParser) {
+    private val operators = expressionParser.getOperators()
+    private val operands = expressionParser.getOperands()
+
+    fun execute(): Int {
+        var result = operands.first()
+
+        operators.forEachIndexed { index, symbol ->
+            val operand2 = operands[index + NEXT_OPERAND_INDEX]
+            result = calculate(symbol, result, operand2)
+        }
+        return result
+    }
+
+    private fun calculate(
+        symbol: String,
+        operand1: Int,
+        operand2: Int,
+    ): Int {
+        return Operator.entries.find { it.symbol == symbol }?.action?.invoke(operand1, operand2)
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls
As suggested in the Kotlin docs, we should put the `.` character or the `?.` operator on the next line, with a single indent.
(The first call in the chain should usually have a line break before it, but it's OK to omit it if the code makes more sense that way.)
```suggestion
        Operator.entries
            .find { it.symbol == symbol }
            ?.action
            ?.invoke(operand1, operand2)
```

Reply:
If there are too many chained calls, this may indicate low cohesion among objects.
I will guide you throughout the RacingCar mission!